hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,373 @@
+"""
+Unit tests for functions in hibike_message.
```

Comment:
maybe change the name of this directory to `hibike_tests` or something similar? I feel like the current name may cause namespace confusion (from the programmer's perspective)

Reply:
Done